dataDispatcher.go

```diff
@@ -2,11 +2,11 @@ package indexer
 
 import (
 	"context"
-	"errors"
+	errorsGo "errors"
```

Comment:
remove this alias (in the whole project). Please use `indexerErrors` alias for `github.com/ElrondNetwork/elastic-indexer-go/errors`

Reply:
done.